package.json

```diff
@@ -2,24 +2,18 @@
   "name": "when-dom-ready",
   "version": "1.2.4",
   "description": "$(document).ready() for the 21st century",
-  "main": "dist/index.js",
+  "main": "dist/es5.umd/index.js",
+  "jsnext:main": "src/index.js",
```

Comment:
This should be `module` instead of `jsnext:main`.

Also, it will still need to be transpiled so arrow fns and Array.includes work in Safari and IE.